main.go

```diff
@@ -207,8 +208,10 @@ func wrappedMain() int {
 			}
 			if backgrounded {
 				fmt.Fprint(os.Stderr, "Running in background, not using a TTY\n")
+				basicUi.PB = &packer.NoopProgressTracker{}
 			} else if TTY, err := openTTY(); err != nil {
 				fmt.Fprintf(os.Stderr, "No tty available: %s\n", err)
+				basicUi.PB = &packer.NoopProgressTracker{}
```

Comment:
I think these two lines should be enough to fix this.

Reply:
Yeah, I think so too. The rest of the code is just to make it so I can actually assign the no op bar to the struct because I don't think I can do that if it's just embedded as a decorator rather than an exported field.